packages/pointer_interceptor/example/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ version: 1.0.0
 
 environment:
   sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.26.0-0" # For integration_test from sdk
```

Comment:
I'm sure this is fine, but do you know why this comment was here before? Has it been resolved?

Reply:
`integration_test` used to be a stand-alone package, but moved into the SDK. That happened at the same time as the NNBD transition was happening, and only the SDK copy was migrated. So we had some cases where (due to transitive dependency conflicts) we needed to use the SDK version before it was in `stable`, so had to require a pre-release version. That's a very unusual thing for us to do, so it had a comment as to why.

It's resolved in the sense that we don't need a pre-release version now, and thus don't need a comment.

Reply:
Thanks for the explanation